qiskit_experiments/library/randomized_benchmarking/rb_analysis.py

```diff
@@ -65,20 +78,22 @@ class RBAnalysis(curve.CurveAnalysis):
         )
     ]
 
+    def __init__(self):
+        super().__init__()
+        self._gate_counts_per_clifford = None
+
     @classmethod
     def _default_options(cls):
         """Default analysis options.
 
         Analysis Options:
-            error_dict (Dict[Tuple[Iterable[int], str], float]): Optional.
-                Error estimates for gates from the backend properties.
-            epg_1_qubit (Dict[int, Dict[str, float]]) : Optional.
-                EPG data for the 1-qubit gate involved,
-                assumed to have been obtained from previous experiments.
-                This is used to estimate the 2-qubit EPG.
-            gate_error_ratio (Dict[str, float]): An estimate for the ratios
-                between errors on different gates.
-
+            gate_error_ratio (Optional[Dict[str, float]]): A dictionary with gate name keys
+                and error ratio values used when calculating EPG from the estimated EPC.
+                The default value will use standard gate error ratios.
+                If set to ``False`` EPG will not be calculated.
```

Comment:
Depends if you want to avoid computation of EPG. If you don't trust gate error ratio then you should be able to disable this to avoid creating some garbage in the database.

Reply:
In that case I think it should be `None` instead of false for the skip case (since you now use "default" for computing defualt)